test/config.json

```diff
@@ -1130,6 +1130,33 @@
 			"RetryMax": 3,
 			"Tags": []
 		},
+		"vtorc_primary_failure_8.0": {
+			"File": "unused.go",
+			"Args": ["vitess.io/vitess/go/test/endtoend/vtorc/primaryfailure"],
+			"Command": [],
+			"Manual": false,
+			"Shard": "vtorc_8.0",
+			"RetryMax": 3,
+			"Tags": []
+		},
+		"vtorc_graceful_takeover_8.0": {
+			"File": "unused.go",
+			"Args": ["vitess.io/vitess/go/test/endtoend/vtorc/gracefultakeover"],
+			"Command": [],
+			"Manual": false,
+			"Shard": "vtorc_8.0",
+			"RetryMax": 3,
+			"Tags": []
+		},
+		"vtorc_general_8.0": {
+			"File": "unused.go",
+			"Args": ["vitess.io/vitess/go/test/endtoend/vtorc/general"],
+			"Command": [],
+			"Manual": false,
+			"Shard": "vtorc_8.0",
+			"RetryMax": 3,
+			"Tags": []
+		},
```

Comment:
I can avoid this duplication of same values of _8.0 by adding code to workflow generator where we remove `_8.0` suffix from the name of the test to find the shard to follow. This would mean that `vtorc_8.0` and `vtorc` will both have the command to follow the shard `vtorc` in the workflow and then we can remove this duplicate code.

I don't have a specific preference between the two, if you feel the other alternate is more readable/better, I'll make the change.

Reply:
This is fine. Naming the test clearly helps in debugging when there are failures in CI.